src/backend/common/helpers/matchstats_helper.py

```diff
@@ -72,184 +191,51 @@ def build_Minv_matrix(
-            stat_type,
-            init_stats=init_stats,
-            init_stats_default=init_stats_default,
-            limit_matches=limit_matches,
+            point_accessor,
         )
-        x = np.dot(Minv, s)
+        x = np.dot(M_inv, s)
 
         stat_dict = {}
         for team, stat in zip(team_list, x):
             stat_dict[team] = stat[0]
-        return stat_dict
-
-    @classmethod
-    def _get_stat(
-        cls,
-        stat_type: StatType,
-        match: Match,
-        alliance_color: AllianceColor,
-        alliance_teams: List[TeamId],
-        init_stats: Optional[Dict[StatType, Dict[TeamId, int]]],
-        init_stats_default: int,
-        treat_as_unplayed: bool,
-    ) -> Optional[int]:
-        match_played = match.has_been_played and not treat_as_unplayed
-
-        if match_played:
-            if stat_type == StatType.OPR:
-                return match.alliances[alliance_color]["score"]
-            elif stat_type == StatType.DPR:
-                if alliance_color == AllianceColor.RED:
-                    other_alliance_color = AllianceColor.BLUE
-                else:
-                    other_alliance_color = AllianceColor.RED
-                return match.alliances[other_alliance_color]["score"]
-            elif stat_type == StatType.CCWM:
-                if alliance_color == AllianceColor.RED:
-                    other_alliance_color = AllianceColor.BLUE
-                else:
-                    other_alliance_color = AllianceColor.BLUE
-                return (
-                    match.alliances[alliance_color]["score"]
-                    - match.alliances[other_alliance_color]["score"]
-                )
 
-        # None of the above cases were met. Return default.
-        if init_stats and stat_type in init_stats:
-            total = 0
-            for team in alliance_teams:
-                total += init_stats[stat_type].get(team, init_stats_default)
-            return total
-        else:
-            total = 0
-            for team in alliance_teams:
-                total += init_stats_default
-            return total
-
-    @classmethod
-    def calculate_matchstats(cls, matches: List[Match], year: Year) -> EventMatchStats:
-        if not matches:
-            return {}
-
-        team_list, team_id_map = cls.build_team_mapping(matches)
-        if not team_list:
-            return {}
-        Minv = cls.build_Minv_matrix(matches, team_id_map, played_only=True)
-
-        oprs_dict = cls.calc_stat(matches, team_list, team_id_map, Minv, StatType.OPR)
-        dprs_dict = cls.calc_stat(matches, team_list, team_id_map, Minv, StatType.DPR)
-        ccwms_dict = cls.calc_stat(matches, team_list, team_id_map, Minv, StatType.CCWM)
-
-        stats: EventMatchStats = {
-            StatType.OPR: oprs_dict,
-            StatType.DPR: dprs_dict,
-            StatType.CCWM: ccwms_dict,
-        }
-
-        return stats
-
-    @classmethod
-    def get_last_event_stats(
```

Comment:
For removing this method - do we just not use this anywhere? I see it was being used in py2 in `calculate_matchstats` - did it get dropped somewhere during the py3 move? https://github.com/the-blue-alliance/the-blue-alliance/blob/1d05f3bb147b26e228efe40801f13a0692a97153/helpers/matchstats_helper.py#L152

Reply:
It's actually an unused variable even in the code you linked, so I'm guessing someone removed that unused variable in the transition, but then the method became unused across the repo.